app-text/htmltidy/tidy57-5.7.0~git.recipe

```diff
@@ -0,0 +1,97 @@
+SUMMARY="The granddaddy of HTML tools, with support for modern standards"
+DESCRIPTION="Tidy tidies HTML and XML. It can tidy your documents by itself, \
+and developers can easily integrate its features into even more powerful tools."
+HOMEPAGE="http://www.html-tidy.org/"
+COPYRIGHT="1998-2015 World Wide Web Consortium"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="28be79db50d2ecf6730068167d8ed23987fa8d59"
+SOURCE_URI="https://github.com/htacg/tidy-html5/archive/$srcGitRev.tar.gz"
+SOURCE_FILENAME="tidy-html5-$portVersion"
```

Comment:
```diff
-SOURCE_FILENAME="tidy-html5-$portVersion"
 CHECKSUM_SHA256="5fddac299a385203e06a2158df3acd68aa83ffd10d5bdf73b9e7f2913dee1a88"
+SOURCE_FILENAME="tidy-html5-$portVersion"
```
Source: [HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering)

Reply:
That's funny, I noticed that SOURCE_FILENAME was not at the right place but I did not notice it was missing the `.tar.gz` suffix! 😲